packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/page-tree-item.tsx

```diff
@@ -41,6 +41,9 @@ const CustomContent = forwardRef((props: TreeItemContentProps, ref) => {
 
   return (
     <Box
+      component={Link}
+      noLinkStyle
+      href={url}
```

Comment:
Wrapping the entire page tree item with a Link means clicking on the expand icon or change icon button will also navigate the page and that isn't a desired behavior


https://user-images.githubusercontent.com/16928688/186387881-61a88b19-481a-4a3f-a4eb-956fedd22351.mov

Reply:
Keyboard navigation is also affected 


https://user-images.githubusercontent.com/16928688/186388513-179dc109-f7d8-4481-bddf-527cada47657.mov

Reply:
Oh – just need to stop event propagation, will do that – we also don't want parts of the hover target to not be clickable